aiohttp/connector.py

```diff
@@ -717,6 +719,7 @@ class TCPConnector(BaseConnector):
         resolver
     use_dns_cache - Use memory cache for DNS lookups.
     ttl_dns_cache - Max seconds having cached a DNS entry, None forever.
+        0 expire immediately (use use_dns_cache=False instead). 10 by default.
```

Comment:
I'd revert this given the above comments. Also the docstring doesn't get output into the docs (You'll see the default value is already mentioned in the actual docs: https://docs.aiohttp.org/en/stable/client_reference.html#tcpconnector).

Reply:
Removed